.npmignore

```diff
@@ -0,0 +1,3 @@
+components/
```

Comment:
Probably a good idea, especially as ES6 (and friends) workflows continue to evolve.